src/modules/Popup/Popup.js

```diff
@@ -0,0 +1,270 @@
+import React, { Component, PropTypes } from 'react'
+import cx from 'classnames'
+import _ from 'lodash'
+import { META } from '../../lib'
+import Portal from '../../addons/Portal/Portal'
+import PopupContent from './PopupContent'
+import PopupHeader from './PopupHeader'
+
+const _meta = {
+  name: 'Popup',
+  type: META.TYPES.MODULE,
+}
+
+
+/**
+ * A Popup displays additional information on top of a page.
+ */
+export default class Popup extends Component {
+  static propTypes = {
+    /** Display the popup without the pointing arrow */
+    basic: PropTypes.bool,
+
+    /** You may pass a content as children of the Popup */
+    children: PropTypes.node,
+
+    /** Classes to add to the Accordion className. */
+    className: PropTypes.string,
+
+    /** Simple text content for the popover */
+    content: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+
+    /** A Flowing popup have no maximum width and continue to flow to fit its content */
+    flowing: PropTypes.bool,
+
+    /** Takes up the entire width of its offset container */
+    fluid: PropTypes.bool,
+
+    /** Header displayed above the content in bold */
+    header: PropTypes.string,
+
+    /** Whether the popup should not close on hover */
+    hoverable: PropTypes.bool,
+
+    /** Invert the colors of the popup */
+    inverted: PropTypes.bool,
+
+    /** The node where the popup should mount.. */
+    hideOnScroll: PropTypes.bool,
+
+    /** Horizontal offset in pixels to be applied to the popup */
+    offset: PropTypes.number,
+
+    /** Event triggering the popup */
+    on: PropTypes.oneOf(['hover', 'click', 'focus']),
+
+    /** Positionning for the popover */
+    position: PropTypes.string,
+
+    /** Popup size */
+    size: PropTypes.string,
+
+    /** custom popup style */
+    style: PropTypes.object,
+
+    /** Element to be rendered in-place where the popup is defined. */
+    trigger: PropTypes.node,
+
+    /** Popup width */
+    width: PropTypes.string,
```

Comment:
There are fixed widths, though I think we should update this to `wide` and `wide='very'` given the SUI classes at hand.

```js
_meta = {
  props: {
    wide: ['very']
  },
}

wide: customPropTypes.some([
  PropTypes.bool,
  PropTypes.oneOf(_meta.props.wide)
]),
```

>Here's more context as to why the rename.  The `width` is primarily used for components that have grid column style widths, or a group component that defines the widths of its children.  It takes a number or a word (`width={4}` `width='four'`).  In this case, there it is more like the `padded` or `relaxed` prop, where it is either boolean or takes a `very` value.